src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -210,6 +212,7 @@ public partial class SslStream : System.Net.Security.AuthenticatedStream
         public virtual void EndAuthenticateAsServer(System.IAsyncResult asyncResult) { }
         public override int EndRead(System.IAsyncResult asyncResult) { throw null; }
         public override void EndWrite(System.IAsyncResult asyncResult) { }
+        ~SslStream() { }
```

Comment:
This one was added to the source when moving code from `SslState` to `SslStream` in [this commit](https://github.com/dotnet/runtime/commit/775eac09c623cb834ffdada8948bb8e43f178bc2). See the [blame](https://github.com/dotnet/runtime/blame/5f2398b1fd864fd988027e35b5cab2b128242720/src/libraries/System.Net.Security/src/System/Net/Security/SslStream.Implementation.cs#L625). Should this be in refs or not? If not, should it be removed from source?
cc: @stephentoub @wfurt

Reply:
It should be in the ref. Without it, if someone were to derive from SslStream and add their own finalizer to the derived type, upon finalization SslStream's finalizer might be skipped.